jobs/JREGIONAL_RUN_MET_ASCII2NC_OBS

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+#
```

Comment:
It could be nice to have a doc block with a description of what this job does.

Reply:
Done, also removed some unnecessary yaml sections.